README.md

```
@@ -0,0 +1,293 @@
+<br>
+
+## 1. Header
+**코드 시작할 때 `<header>`로 시작하는 것처럼 소상공인들의 디지털화의 시작을 응원한다는 마음을 담아 Header라 칭함**
+
+<br>
+
+## 2. Team Members Information
+
+| Project Manager | Configuration Manager | Database Manager | Database Manager |
+|-------------------|-------------------|-------------------|-------------------|
+| ![윤정아](https://github.com/user-attachments/assets/46ed801d-74f0-4a57-8d2e-ec7e62425b82)  | ![서열익](https://github.com/user-attachments/assets/d75f9846-532c-480f-b255-75fa342ed5b2)  | ![김예람](https://github.com/user-attachments/assets/1184360e-376d-42a1-a07d-f3226fc3a8de)  | ![김주혜](https://github.com/user-attachments/assets/3fda8e72-8f3a-4fdf-828c-103d97d7fd6c)
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)